build.gradle

```diff
@@ -107,10 +107,11 @@ allprojects {
             'bd0f0dd9-1356-4dec-a8a3-edc777cc92d5', 'b70e447a-06a1-48b0-a02b-cd5b78862777',
 
             //pkg:maven/org.apache.logging.log4j/log4j-core@2.11.2 (from org.scala-sbt:zinc_2.12:1.3.5 - Used by gradle scala plugin only)
-            'd3477f9c-032a-44a7-a5e1-02ae35e4737c',
+            'd3477f9c-032a-44a7-a5e1-02ae35e4737c', '9e818913-69a3-41c8-9bcc-6293b378c53a', 'f0ac54b6-9b81-45bb-99a4-e6cb54749f9d', '7cc258a5-d3ab-451f-bd27-415ae0e3b457',
+            'edaf092e-e7f3-4c69-8f01-a5c6fc44890a',
 
             //pkg:maven/log4j/log4j@1.2.17 (from com.vladsch.flexmark:flexmark-all:0.35.10 - Used in Unit Test runs only)
```

Comment:
We should have looked at upgrading this library. Don't worry about it now, but let's try to remember in the future.